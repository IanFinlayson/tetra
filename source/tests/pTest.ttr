def main():
   x = 4
   print("In main(), just set x = 4 and x = ", x, "\n")
   y = 4
   print("In main(), just set y = 4 and y = ", y, "\n")
   if x == y:
      print("x = y\n")
      print("x + y = ", x+y,"\n")
      print("x / y = ", x/y,"\n")
      print("x * y = ", x*y,"\n")
      print("x - y = ", x-y,"\n")
      print("Performed unstored operations and x = ", x, " and y = ", y, "\n")
      z = x + x
      print("Just set z = x + x and z = ", z, "\n")
      x = 2
      print("Just set x = 2 and x = ", x, "\n")
      y = 3
      print("Just set y = 3 and y = ", y, "\n")
      x = x + y
      print("Just set x = x + y and x = ", x, "\n")
      y = y + x
      print("Just set y = y + x and y = ", y, "\n")
