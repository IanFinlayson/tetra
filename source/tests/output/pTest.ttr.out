In main(), just set x = 4 and x = 4
In main(), just set y = 4 and y = 4
x = y
x + y = 8
x / y = 1
x * y = 16
x - y = 0
Performed unstored operations and x = 4 and y = 4
Just set z = x + x and z = 8
Just set x = 2 and x = 2
Just set y = 3 and y = 3
Just set x = x + y and x = 5
Just set y = y + x and y = 8

